### ISSUE TITLE
Input fields in modal forms stop responding in IE11

### ISSUE THREAD

**Issue by robguthrie** [loomio/loomio#3805]
We had a bug report from an IE11 user that the "Are you sure you want to be on free plan" modal had stopped responding.

In trying to confirm that I found that the forms in IE11 stop working at some point, and the placeholder text does not go away on focus.. you can see here that when I try to focus the field it just highlights the word of the disabled placeholder text:

![image](https://cloud.githubusercontent.com/assets/486367/19535333/ae6bc4d6-96a3-11e6-9440-bde6fcc51dbc.png)

**Comment by robguthrie** [comment]
I've looked at the developer error console and cannot see any issues.

**Comment by robguthrie** [comment]
Also note that the height of the modal compared to the viewport is rather unsatisfying.

**Comment by gdpelican** [comment]
I've upgraded angular material, but can't get browserling to do any of the things described here on IE 10 or 11.

Basically I think we need to invest in an array of testing devices if we're going to actually feel confident about our X-browser compat; I don't trust browserling enough to be able to resolve issues like this.

**Comment by impishj** [comment]
https://github.com/magnetikonline/linuxmicrosoftievirtualmachines

The above with virtual box is pretty reliable, I only ran into strange issues when a user had some toolbar or other malware like thing messing with their system.

IMHO support on windows/OS X might be better achieved through an electron app because IE will waste so much dev time, and occasionally safari too. I spent a year in which it seemed most of my time was chaseing bugs, although it was all the way back to IE 8 so awful :p

I would guess that added dev time is why slack has an app and not just a web view. Although they might have file IO reasons as well not that into their platform
